services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -102,6 +103,26 @@ func releasesDirectoryWithVersion(fs billy.Filesystem, application string, versi
 	return fs.Join(releasesDirectory(fs, application), versionToString(version))
 }
 
+// releasesDirectoryWithVersion returns applications/<app>/releases/<version>
+func releasesDirectoryWithVersionNumber(fs billy.Filesystem, application string, version string) string {
+	return fs.Join(releasesDirectory(fs, application), version)
+}
+
+func (s *State) checkIfReleaseDirectoryExists(fs billy.Filesystem, application string, version types.ReleaseNumbers) (string, error) {
```

Comment:
the function name is a bit misleading,
maybe something like "checkWhichVersionDirectoryExists"?